### PR TITLE
Refresh install/other/rancher.

### PR DESCRIPTION
**Pull request by jhkrug**
There are some TODOs requiring attention before the is ready to merge.

### CODE REVIEW

File: docs/installation/other_inst_scenarios/install_epinio_on_rancher.md [hunk 1]
```diff
@@ -1,40 +1,51 @@
...
+
+:::note
 
-> NOTE: If you are using the rancher dashboard in the browser, it is likely you already have an Ingress controller running.
+If using the Rancher dashboard in a browser, it's likely you already have an Ingress controller running.
 
-Go to `Apps` > `Repositories` > `Create`.
+:::
+
+<!--TODO: This is for when in the Rancher dashboard, right?-->
```
Comment (andreas-kupries) [inline docs/installation/other_inst_scenarios/install_epinio_on_rancher.md:41]: Based on the following text I believe that this is true.

**Review by andreas-kupries — COMMENTED** [review]
Just a comment, and no answers to the tech questions.

File: docs/installation/other_inst_scenarios/install_epinio_on_rancher.md [hunk 2]
```diff
@@ -1,40 +1,51 @@
...
-* Rancher 2.7.1
+- [epinio helm chart 1.7.1](https://github.com/epinio/helm-charts/releases/tag/epinio-1.7.1)
+- Rancher 2.7.1
 
 ## Kubernetes cluster
 
-The two supported methods are to either deploy a new cluster or to import an existing one into Rancher. The steps to take for either method are in the Rancher documentation (See `Cluster Provisioning`).<br />
-*While it is possible (for development) to deploy Epinio on the same node where Rancher is installed, this requires more steps and is not supported by Rancher.*
+The two supported methods are: to deploy a new cluster or to import an existing one into Rancher.
+<!--TODO:A direct link would be better if possible. But I can't seee an obvious one.-->
```
Comment (andreas-kupries) [inline docs/installation/other_inst_scenarios/install_epinio_on_rancher.md:21]: Original text by @juadk . Maybe he remembers where he looked.

File: docs/installation/other_inst_scenarios/install_epinio_on_rancher.md (hunk 1 above)
Comment (jhkrug) [reply docs/installation/other_inst_scenarios/install_epinio_on_rancher.md]: Thanks for confirming.

**Review by andreas-kupries — APPROVED** [review]

**Pull request merged**